standards/templates/standards/curriculum_nogrid.html

```diff
@@ -59,15 +59,34 @@ <h2 class="disclaimer">Standards Alignment</h2>
         {% endif %}
     {% endif %}
     </div>
-
+    {% if curriculum.is_csf%}
+      <div>
+        <h2>How should I use this information?</h2>
+        <p>
+          <span style="font-weight:900;">CSTA Standards Alignment</span>: Code.org’s CS Fundamentals course is aligned with the Computer Science Teachers’ Association (CSTA) standards for K-5 students. You can hit these standards by following our lesson plans. You can also see what CSTA standards your class has worked on in the Standards view of the teacher dashboard. <a href="https://support.code.org/hc/en-us/articles/360041726272-Can-I-see-my-class-s-progress-on-CSTA-Standards-Is-there-an-easy-progress-report-I-can-send-out-for-my-class-">Learn more.</a>
+        </p>
+        <a class="btn" role="button" href="{% url 'curriculum:by_curriculum_csv' curriculum.slug %}">
+            <span class="glyphicon glyphicon-download-alt" aria-hidden="true"></span> {% trans 'Download standards mapping as CSV' %}
+        </a>
+        <p>
+          <span style="font-weight:900;">Cross-Curricular Standards</span>: Computer science helps students with problem-solving, logical thinking, cause and effect, and computational thinking. CS Fundamentals also includes opportunities for teachers to reinforce concepts related to English language arts, mathematics, and science while students are learning computer science. You can help students practice these standards just by following our lesson plans or by making slight adaptations, as outlined in our Cross-curricular Standards Guidance below.
```

Comment:
super tiny: at the end of this line, it says "Cross-curricular" instead of "Cross-Curricular"